browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -7002,5 +7004,17 @@ L.MessageStore = L.Class.extend({
 		Object.keys(othersMessages).forEach(function (msgType) {
 			othersMessages[msgType].forEach(callback);
 		});
+	},
+
+	_cleanUpSelectionMessages: function(messages) {
+		// must be called only from _replayPrintTwipsMsg !!
```

Comment:
is that comment valid?
I see it's called just few lines above not in any `_replayPrintTwipsMsg`
(or do you mean that code above is called from `_replayPrintTwipsMsg` - I didn't check the stack...)

Reply:
Yes, forEach is part of _replayPrintTwipsMsg :)